src/pages/HappDetails/HappDetails.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react'
+import HappThumbnail from 'components/HappThumbnail'
+import HostButton from 'components/HostButton'
+import Button from 'components/Button'
+import Modal from 'components/Modal'
+import './HappDetails.module.css'
+
+import { useQuery, useMutation } from '@apollo/react-hooks'
+import AllAvailableHappsQuery from 'graphql/AllAvailableHappsQuery.gql'
+import EnableHappMutation from 'graphql/EnableHappMutation.gql'
+import DisableHappMutation from 'graphql/DisableHappMutation.gql'
+const findAppByID = id => happ => happ.id === id
+
+export default function BrowseHapps ({
+  history: { push } = {},
+  match: { params } = {}
+} = {}) {
+  const { data: { allAvailableHapps = [] } } = useQuery(AllAvailableHappsQuery)
```

Comment:
could instead make a HappQuery that takes the id and just returns the Happ we want. In this case it def doesn't save loc, but I like it better for saying more exactly what we mean.